Code-250/my-brand
# qXDzq0Iy-portfolio-page

### What does this PR do?

This Pull Request Implements the Portfolio page of my-brand project with its responsiveness.

### Description of Task to be completed?

- should create a grid of the projects list
- should be responsive in all devices
- on hover, it should show the title of the project and buttons one to redirect to live project and the other to redirect to the source codes of the project

### How should this be manually tested?

- `git clone git@github.com:Code-250/my-brand.git` to have the project in your local computer.
- git checkout `ft-portfolio-page-qXDzq0Iy`
- run `index.html` to see test the branch.

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- [portfolio-page](https://trello.com/c/qXDzq0Iy/3-portfolio-page)

### Screenshots (if appropriate)
- Desktop View
 
![desktop](https://user-images.githubusercontent.com/47631065/146904688-6ba868df-4d78-4413-8136-99a446ac212a.png)
- Mobile view
![Mobile](https://user-images.githubusercontent.com/47631065/146904968-10e5b7bd-f557-42fa-b509-15811d7403de.png)

### Questions:
N/A
